app/views/pages/updating-information.njk

```diff
@@ -0,0 +1,55 @@
+{% set pageTitle = "Updating information on this site" %}
+{% set pageDescription = "Find out about updating information on this site." %}
+
+{% extends "includes/layout.njk" %}
+
+{% block outerBody %}
+  <section class="nhsuk-hero">
+    <div class="nhsuk-width-container nhsuk-hero--border">
+      <div class="nhsuk-grid-row">
+        <div class="nhsuk-grid-column-two-thirds">
+          <div class="nhsuk-hero__wrapper">
+            <h1 class="nhsuk-u-margin-bottom-3">{{ pageTitle }}</h1>
+            <p class="nhsuk-body-l nhsuk-u-margin-bottom-0">{{ pageDescription }}</p>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <nav class="nhsuk-breadcrumb" aria-label="Breadcrumb">
+    <div class="nhsuk-width-container">
+      <ol class="nhsuk-breadcrumb__list">
+        <li class="nhsuk-breadcrumb__item">
+          <a class="nhsuk-breadcrumb__link" href="/">Home</a>
+        </li>
+      </ol>
+      <p class="nhsuk-breadcrumb__back">
+        <a class="nhsuk-breadcrumb__backlink" href="/">Home</a>
+      </p>
+    </div>
+  </nav>
+{% endblock %}
+
+
+{% block body %}
+  <div class="nhsuk-grid-row">
+    <div class="nhsuk-grid-column-three-quarters">
+      <p>This site has been made using the NHS.UK Service Manual guidelines.</p>
```

Comment:
@BigEdDev this should be "the NHS digital service manual" 

The name and capitalisation of digital is important here